src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -92,6 +92,7 @@
  * <tr><td>switch ([expr]) { case : [case-range] default: [default-range] }</td>
  * <td>S(i=1:i=n)NP(case-range[i]) + NP(default-range) + NP(expr)</td></tr>
  * <tr><td>[expr1] ? [expr2] : [expr3]</td><td>NP(expr1) + NP(expr2) + NP(expr3) + 2</td></tr>
+ * <tr><td> when [expr] </td><td>NP(expr) + 1</td></tr>
```

Comment:
We don't need space before `when` and after `[expr]`.

I do find this odd that it is separated from the switch, especially it not coming directly after the switch's tr. It should be moved up one line at the least.

Reply:
Moved it to be just below switch. I didn't want to keep it in the same row with switch to not make expression complicated and hard to read.

Reply:
Yea, I understood that, but then there is nothing that says this is for a switch which is what makes it odd for me. I am ok with this until Java uses `when` in a second location.

Reply:
@mahfouz72 Not done.
> We don't need space before when and after [expr].

Reply:
@rnveach done